ui/src/utils/randomUtils.ts

```diff
@@ -6,7 +6,7 @@
 import * as crypto from "crypto";
 
 export namespace randomUtils {
-    export function getPseudononymousStringHash(s: string, encoding: crypto.HexBase64Latin1Encoding = 'base64'): string {
+    export function getPseudononymousStringHash(s: string, encoding: crypto.BinaryToTextEncoding = 'base64'): string {
```

Comment:
Why was this changed?

Reply:
I think they just changed the name of the type between Node 12 and Node 14